x-pack/plugins/security_solution/public/cases/components/add_comment/index.test.tsx

```diff
@@ -6,42 +6,24 @@
 
 import React from 'react';
 import { mount } from 'enzyme';
+import { waitFor, act } from '@testing-library/react';
 
 import { AddComment, AddCommentRefObject } from '.';
 import { TestProviders } from '../../../common/mock';
-import { getFormMock } from '../__mock__/form';
 import { Router, routeData, mockHistory, mockLocation } from '../__mock__/router';
 
 import { CommentRequest, CommentType } from '../../../../../case/common/api';
-import { useInsertTimeline } from '../../../timelines/components/timeline/insert_timeline_popover/use_insert_timeline';
+import { useInsertTimeline } from '../use_insert_timeline/use_insert_timeline';
```

Comment:
nit -> instead of naming the file `use_insert_timeline`, we can just name it `index.tsx`. Therefore we can avoid the duplication in the path.